cocotb/share/lib/vpi/VpiImpl.cpp

```diff
@@ -72,6 +72,26 @@ void VpiImpl::get_sim_precision(int32_t *precision)
     *precision = vpi_get(vpiTimePrecision, NULL);
 }
 
+const char *VpiImpl::get_simulator_product()
+{
+    if (m_product.empty()) {
+        s_vpi_vlog_info info;
+        vpi_get_vlog_info(&info);
+        m_product = info.product;
```

Comment:
The VHPI implementation releases the handle, do we need to do that here? The original code did not.